cli/command/cli.go

```diff
@@ -137,7 +142,28 @@ func (cli *DockerCli) ServerInfo() ServerInfo {
 
 // ClientInfo returns the client details for the cli
 func (cli *DockerCli) ClientInfo() ClientInfo {
-	return cli.clientInfo
+	if cli.clientInfo != nil {
+		return *cli.clientInfo
+	}
+	var experimentalValue string
+	// Environment variable always overrides configuration
+	if experimentalValue = os.Getenv("DOCKER_CLI_EXPERIMENTAL"); experimentalValue == "" {
+		experimentalValue = cli.ConfigFile().Experimental
+	}
+	hasExperimental, _ := isEnabled(experimentalValue)
```

Comment:
Here we are dropping the error, so we will be silent if the `experimentalValue` is not valid. I don't want to panic here, but returning an error in `ClientInfo()` would break many people. @thaJeztah any idea?